src/redux/rockets/rocketSlice.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable array-callback-return */
```

Comment:
Kindly go through your project and remove all lines that disable linter checks using the eslint disable statement. The linter checks are setup to help us learn best coding practices, and solving linter errors yourself will help you become a better developer😎

Reply:
Hi @Shisui6, Thanks for the constructive review, we will remove them.